commit 6f6ee3c5b2ad
Author: Kearwood Gilbert <kgilbert@mozilla.com>
Date:   Wed Oct 11 14:25:51 2017 -0700

    Bug 1402871 - Change VREventObserver::mWindow to a RefPtr. r=dveditz, a=ritu
    
    MozReview-Commit-ID: 79MlNMAvIQD
    
    --HG--
    extra : source : 84ff58454e222ce067ec27f8841ba684934b7666
---
 dom/vr/VREventObserver.h | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git dom/vr/VREventObserver.h dom/vr/VREventObserver.h
index 24a1065a18fe..cd2cf69469b5 100644
--- dom/vr/VREventObserver.h
+++ dom/vr/VREventObserver.h
@@ -34,8 +34,7 @@ public:
 private:
   ~VREventObserver();
 
-  // Weak pointer, instance is owned by mWindow.
-  nsGlobalWindow* MOZ_NON_OWNING_REF mWindow;
+  RefPtr<nsGlobalWindow> mWindow;
   // For WebVR telemetry for tracking users who view content
   // in the 2D view.
   TimeStamp mSpendTimeIn2DView;
